x-pack/plugins/security_solution/public/dashboards/hooks/use_create_security_dashboard_link.test.tsx

```diff
@@ -6,36 +6,56 @@
  */
 
 import { renderHook, act } from '@testing-library/react-hooks';
-import type { DashboardStart } from '@kbn/dashboard-plugin/public';
 import { useKibana } from '../../common/lib/kibana';
 import { useCreateSecurityDashboardLink } from './use_create_security_dashboard_link';
 import { DashboardContextProvider } from '../context/dashboard_context';
 import { getTagsByName } from '../../common/containers/tags/api';
-
-jest.mock('../../common/lib/kibana');
+import React from 'react';
+import { NavigationProvider } from '@kbn/security-solution-navigation';
+import { coreMock } from '@kbn/core/public/mocks';
+import { TestProviders } from '../../common/mock';
+
+jest.mock('../../common/lib/kibana', () => ({
+  useKibana: jest.fn(),
+}));
 jest.mock('../../common/containers/tags/api');
-const URL = '/path';
+jest.mock('../../common/lib/apm/use_track_http_request');
+jest.mock('../../common/components/link_to', () => ({
+  useGetSecuritySolutionUrl: jest
+    .fn()
+    .mockReturnValue(jest.fn().mockReturnValue('/app/security/dashboards/create')),
+}));
 
 const renderUseCreateSecurityDashboardLink = () =>
   renderHook(() => useCreateSecurityDashboardLink(), {
-    wrapper: DashboardContextProvider,
+    wrapper: ({ children }) => (
+      <TestProviders>
+        <NavigationProvider core={coreMock.createStart()}>
```

Comment:
NIT: there's a mock available if you don't want to add the navigation provider.
`jest.mock('@kbn/security-solution-navigation/src/context');`